homeassistant/components/myuplink/switch.py

```diff
@@ -21,7 +21,11 @@
     "NIBEF": {
         "50004": SwitchEntityDescription(
             key="temporary_lux",
-            icon="mdi:water-alert-outline",
+            translation_key="temporary_lux",
+        ),
+        "50005": SwitchEntityDescription(
+            key="boost_ventilation",
+            translation_key="boost_ventilation",
         ),
```

Comment:
This looks like a new entity is added? It would be good if this could be moved into a separate/follow-up PR.
(Feel free to ping me for reviewing such a small follow-up PR)

Reply:
No, it is not a new entity. It was generated automatically from metadata in the API as 95% of all entities in this integration. But since I want to assign an specific icon it is necessary to create a tiny SwitchEntityDescription. If this change is moved to a separate PR, corresponding icon translation should also be moved? Or?

Reply:
Ah, I see. Since this is just used to add the translation key then, it's fine IMO.